spring-cloud-gcp-vision/src/main/java/org/springframework/cloud/gcp/vision/CloudVisionTemplate.java

```diff
@@ -58,13 +77,20 @@ public CloudVisionTemplate(ImageAnnotatorClient imageAnnotatorClient) {
 	 * feature types can be found in {@link Feature.Type}.
 	 * @param imageResource the image one wishes to analyze. The Cloud Vision APIs support
 	 *     image formats described here: https://cloud.google.com/vision/docs/supported-files
-	 * @param featureTypes the types of image analysis to perform on the image.
+	 * @param featureTypes the types of image analysis to perform on the image
 	 * @return the results of image analyses
+	 * @throws CloudVisionException if the image could not be read or if a malformed response
```

Comment:
```suggestion
	 * @throws {@link CloudVisionException} if the image could not be read or if a malformed response
```

Reply:
Actually I don't think these suggestions work. When I add links I get the checkstyle error: 

> Javadoc comment at column 13 has parse error. Details: no viable alternative at input '{' while parsing JAVADOC_TAG [NonEmptyAtclauseDescription]

I think for exceptions you can omit the `@link`?